### PR TITLE
"Modify s_h_a to allow puppet-core pre-commit testing"

### PR DESCRIPTION
**Pull request by nibalizer**

### CODE REVIEW

File: spec/spec_helper_acceptance.rb [hunk 1]
```diff
@@ -34,7 +56,7 @@
 
     # Install module and dependencies
     hosts.each do |host|
-      copy_module_to(host, :source => proj_root, :module_name => 'apache')
+      puppet_module_install(:source => proj_root, :module_name => 'apache', :target_module_path => '/etc/puppetlabs/code/modules')
```
Comment (bmjen) [inline spec/spec_helper_acceptance.rb]: :+1:  been on my to-do for a long time....

**Comment by bmjen** [comment]
Whats with the pcci failures?

File: spec/spec_helper_acceptance.rb (hunk 1 above)
Comment (bmjen) [inline spec/spec_helper_acceptance.rb]: @nibalizer I think you need to remove the `target_module_path` option.

**Comment by igalic** [comment]
"why is @nibalizer writing quoted commit messages?"

**Comment by nibalizer** [comment]
This entire thing is a secret project. Move along.

**Comment by nibalizer** [comment]
@igalic my pull-requests come from this: https://github.com/nibalizer/dotfiles/blob/master/local/bin/git-pull-request improving it improves my pull-requets

**Comment by jonnytpuppet** [comment]
Hi There

Before this can be merged, the tests need to pass
Thanks

**Comment by bmjen** [comment]
Hey @nibalizer, any updates on this PR?

**Comment by hunner** [comment]
I believe there is actually a beaker `git` installation method for puppet, though I've never used it. It's based on the nodeset parameters.

**Pull request closed without merging**